modules/proc_configuring-notificationsconfiguration-crd-by-using-the-ocp-cli.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-notificationsconfiguration-cr-by-using-cli_{context}"]
+= Configuring the NotificationsConfiguration CR by using the CLI
+
+You can configure the `NotificationsConfiguration` custom resource (CR) by using the CLI (`oc`).
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance. For more information, see "Enabling notifications with an Argo CD instance".
+
+.Procedure
+
+. Enable notifications for the Argo CD instance in the CLI by running the following command:
++
+[source,terminal]
+----
+$ oc edit <argo_CD_instance> -n <namespace>
+----
++
+where:
+
+`argocd openshift-gitops`:: Specifies the name of the Argo CD instance.
+`openshift-gitops`:: Specifies the name of the namespace.
```

Comment:
You still need to change these values to match what's in the command:
```suggestion
`<argo_CD_instance>`:: Specifies the name of the Argo CD instance.
`<namespace>`:: Specifies the name of the namespace.
```

Reply:
Fixed.